### DISCUSSION TITLE
No input normalization?

### DISCUSSION THREAD

**Discussion by nirleibushor** [STVIR/pysot#448] [Code]
Hi,

Going through the code I noticed that the input images are given to the model as-is, in BGR values of 0-255, with no normalization.

Is it really so or I missed something?

Thanks

**Comment by ZhiyuanChen** [comment]
It is really so.

**Reply by nirleibushor** [reply]
Is there a reason that there is no normalization?
Just wondering because it isn't standard in deep-learning as far as I know.

**Reply by ZhiyuanChen** [reply]
That could be a great question which only @lb1100 could answer
